cmd/build-helm.go

```diff
@@ -97,5 +100,12 @@ func init() {
 		"Additional information to use in computing the image tags",
 	)
 
+	buildHelmCmd.PersistentFlags().BoolP(
+		"use-secrets-generator",
+		"",
+		false,
+		"The project uses a secret generator pre-flight job",
```

Comment:
I have problems creating a short description what it does beyond "preserve the old functionality before we had secrets generator roles".

Some of the things it does:

* no longer generates passwords in the kube secrets
* doesn't include default settings for generated secrets other than passwords (SSH keys and SSL certs and CAs) in `values.yaml`
* calls the secret object `secrets-update` instead of `secrets` but still references the `secret` object from all other objects

I would rather document these requirements in the `README.md` of the `fissile-secrets-generator` repo. I think it is too much detail for a CLI option summary.

But anyways, what would be a better name for this option?

Reply:
> doesn't include default settings for generated secrets other than passwords

Ok, I can barely understand what I wrote myself; what I meant was:

* includes the names of generated secrets in `values.yaml`, but ignores the values provided by `--defaults-file`
* this includes the names of generated password, which aren't user-overridable without this option

Reply:
Best name I can think of is `--skip-generated-secrets`, but that's not _really_ better.

I think I'm okay with leaving it as `--use-secrets-generator` if the help text explains how to use it (or point to a place that tells you how to use it, given the verbosity).